elements/lisk-tree/src/merkle_tree.ts

```diff
@@ -522,15 +382,15 @@ export class MerkleTree {
 		const nodeValue = this._hashToValueMap[hashValue.toString('binary')];
 
 		if (isLeaf(nodeValue)) {
-			return nodeValue.toString('base64');
+			return hashValue.toString('hex');
```

Comment:
please use `base64` for binary string format for consistency

Reply:
It was actually inconsistent before, since it was printing the branches as hex and only the leaves as base64. This way it is consistent and useful for debugging.